src/uu/cp/src/cp.rs

```diff
@@ -1586,6 +1612,20 @@ fn copy_file(
         return Ok(());
     }
 
+    // issue 5031
```

Comment:
This is context that loses its meaning very quickly. An explanation of the issue would be more helpful than the id (but I think it's ok to not reference it at all, the code makes a lot of sense now).